doc/source/user_guide/hpc.rst

```diff
@@ -0,0 +1,16 @@
+
+.. _ref_hpc:
+
+
+===============================
+High performance clusters (HPC)
```

Comment:
```suggestion
High performance clusters (HPC)
```
HPC typically stands for high performance computing. Should we be using it here for high performance clusters?

Reply:
Oh my... you are right... as always :)